ModernFlyouts/Controls/AirplaneModeControl.xaml

```diff
@@ -1,19 +1,18 @@
 ﻿<UserControl x:Class="ModernFlyouts.AirplaneModeControl"
              xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
              xmlns:ui="http://schemas.modernwpf.com/2019"
              xmlns:utils="clr-namespace:ModernFlyouts.Utilities"
-             xmlns:resx="clr-namespace:ModernFlyouts.Properties"
+             xmlns:resx="clr-namespace:ModernFlyouts.Properties" xmlns:modernflyouts="clr-namespace:ModernFlyouts" d:DataContext="{d:DesignInstance Type=modernflyouts:AirplaneModeFlyoutHelper}"
```

Comment:
No need to set the design time datacontext as the flyout will only be initialized at runtime.

There is a check in the helpers to design time and not do anything.
This will just make the xaml complex. No use. Revert it